src/dom/Draggable.js

```diff
@@ -39,9 +39,11 @@ L.Draggable = L.Evented.extend({
 		}
 	},
 
-	// @constructor L.Draggable(el: HTMLElement, dragHandle?: HTMLElement, preventOutline: Boolean)
+	// @constructor L.Draggable(el: HTMLElement, dragHandle?: HTMLElement, preventOutline?: Boolean, options?: Draggable options)

```

Comment:
@nathancahill Please add a `@aka` directive so that `Draggable options` will actually become a link to the class' options section, see https://github.com/Leaflet/Leaflet/blob/master/src/layer/Popup.js#L41